### ISSUE TITLE
Reasonable adaptations and use cases? 

### ISSUE THREAD

**Issue by sgt101** [nestauk/ojd_daps_skills#234]
Hi there, 

I am working on a project to extract skills from CV's - so kind of the inverse of extracting them from job ads. 

I want to use your mapper to map to a standard taxonomy so that there is consistency. 

Does this feel like a reasonable adaptation to you guys? 

Also,  I guess one of my questions would be about how to interact with the standard taxonomies - do they have APIs or do we need to subscribe/licence, etc?

**Comment by lizgzil** [comment]
Hi @sgt101 - thanks for your interest in our library.

The skill extraction model was trained to extract skills from job advert text, so we don't know how well it would perform well on CVs where the language style and format is different. But nothing to stop you trying it out and see how you feel it did! I'd encourage you to evaluate some of the results though.

Am I right that your input is a data set of CVs and the output is the skills in them standardised to either the ESCO or the Lightcast taxonomies?

If so, you don't need to subscribe to the taxonomies or anything, you can just use the instructions outlined in the main README of this repo (changing `taxonomy_name="toy"` to `taxonomy_name="esco"` or `taxonomy_name="lightcast"` depending on which taxonomy you want to map to).

If you already have a list of skills and want to map them only, you can use the code I added to [this issue](https://github.com/nestauk/ojd_daps_skills/issues/233)

Let me know if I've interpreted that incorrectly!

**Comment by sgt101** [comment]
Yes - that's 100% what we are thinking. I completely agree about the evaluation and we will definitely try things out. It's very useful that we could ask and check that this is not just a silly idea though - thanks for taking the time to read, respond and make such a helpful suggestion. 

I will let you know how we get on.

**Comment by sherlach** [comment]
@sgt101 Did you end up making any progress in this use-case? I'm looking at a similar project to extract skills from resumes, curious if you found any success here
